### ISSUE TITLE
Error: docker-compose: command not found

### ISSUE THREAD

**Issue by sergeycherepanov** [iam4x/bobarr#273]
Install script does not support modern docker compose

The error:
```
bash: line 52: docker-compose: command not found
```

```
root@ubuntu-2gb-hel1-1:~/bobarr# docker compose version
Docker Compose version v2.18.1
```

**Comment by Oratorian** [comment]
edit the bobarr.sh and modify docker-compose to docker compose save, then do bash bobarr.sh

**Comment by iam4x** [comment]
Closed with PR #276
